manifests/params.pp

```diff
@@ -35,6 +35,12 @@
       $servicename      = 'dhcpd'
       $package_provider = undef
     }
+    'Archlinux': {
+      $dhcp_dir         = '/etc'
```

Comment:
He does!
I'm not sure how I want to workaround the broken defaults from upstream. The prefered way seems to be to throw everything in one file, called /etc/dhcpd.conf. The default for other distributions is to put the content into seperate files in /etc/dhcp/...